nextjs-app/components/pages/program-rules/ProgramFees.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC } from "react";
+import Image from "next/image";
+import SectionTitle from "./SectionTitle";
+
+const FEE_DATA: FeeInfo[] = [
```

Comment:
```suggestion
const FEE_DATA = [
```

We don't need to specify the type